File: victorious/victorious/UI/Common/Action Bar/VActionBarFlexibleSpaceItem.m
//
//  VActionBarFlexibleSpaceItem.m
//  victorious
//
//  Created by Michael Sena on 4/14/15.
//  Copyright (c) 2015 Victorious. All rights reserved.
//

#import "VActionBarFlexibleSpaceItem.h"

@implementation VActionBarFlexibleSpaceItem

+ (VActionBarFlexibleSpaceItem *)flexibleSpaceItem
{
    VActionBarFlexibleSpaceItem *flexibleSpaceItem = [[VActionBarFlexibleSpaceItem alloc] initWithFrame:CGRectZero];
    flexibleSpaceItem.translatesAutoresizingMaskIntoConstraints = NO;
    return flexibleSpaceItem;
}

@end

File: victorious/victorious/UI/Common/Action Bar/VActionBarFlexibleSpaceItem.swift
//
//  VActionBarFlexibleSpaceItem.swift
//  victorious
//
//  Created by Sharif Ahmed on 3/15/16.
//  Copyright © 2016 Victorious. All rights reserved.
//

import Foundation

class VActionBarFlexibleSpaceItem: UIView {
    
    class func flexibleSpaceItem() -> VActionBarFlexibleSpaceItem {
        let flexibleSpaceItem = VActionBarFlexibleSpaceItem.init(frame: CGRect.zero)
        flexibleSpaceItem.translatesAutoresizingMaskIntoConstraints = false
        return flexibleSpaceItem
    }
}
